helm/charts/hydra-maester/templates/pdb.yaml

```diff
@@ -10,5 +10,9 @@ spec:
       control-plane: controller-manager
       app.kubernetes.io/name: {{ include "hydra-maester.fullname" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
-  {{ toYaml .Values.pdb.spec }}
+  {{- if .Values.pdb.spec.maxUnavailable }}
+  maxUnavailable: {{ .Values.pdb.spec.maxUnavailable }}
+  {{- else }}
+  minAvailable: {{ .Values.pdb.spec.minAvailable | default "1" }}
```

Comment:
i dont like this hidden default value, maybe the following form would be more explicit?

```yaml
# values.yaml
pdb:
  enabled: true
  spec:
    maxUnavailable: ""
    minAvailable: ""
---
# template.yaml
{{- with .Values.pdb.spec.maxUnavailable }}
maxUnavailable: {{ . }}
{{- end }}
{{- with .Values.pdb.spec.minAvailable }}
minAvailable: {{ . }}
{{- end }}
```

I know the spec allows for only one field to be set, but `with` won't render empty fields, and imho it is the users responsibility to give sane data to the system